includedBuild/gradleConfiguration/src/main/kotlin/com/badoo/reaktive/configuration/MppConfigurationPlugin.kt

```diff
@@ -62,6 +62,9 @@ class MppConfigurationPlugin : Plugin<Project> {
                 maybeCreate("jvmJsCommonMain").dependsOn(getByName("commonMain"))
                 maybeCreate("jvmJsCommonTest").dependsOn(getByName("commonTest"))
 
+                maybeCreate("jsWasmJsCommonMain").dependsOn(getByName("commonMain"))
```

Comment:
I suggest to name it just `jsCommonMain` and `jsCommonTest`. It should also depend on `jvmJsCommonMain`.

Reply:
Done!